.github/workflows/slack.yml

```diff
@@ -0,0 +1,18 @@
+name: Notify on merged PR
+on:
+    pull_request:
+        types:
+            - closed
+
+jobs:
+    notify_merged_pr:
+        name: Notify on merged PR
+        runs-on: ubuntu-22.04
+        if: ${{ github.event.pull_request.merged == true }}
+        steps:
+            - uses: abinoda/slack-action@master
+              env:
+                  SLACK_BOT_TOKEN: ${{ secrets.VIZZUHQ_SLACK_API }}
+              with:
+                  args: '{\"channel\":\"${{ secrets.VIZZUHQ_SLACK_CHANNEL }}\",\"blocks\":[{\"type\":\"section\",\"text\":{\"type\":\"mrkdwn\",\"text\":\"*PR:* ${{ github.event.pull_request.title }}\n*Description :* ${{ github.event.pull_request.body }}\n*Merged at:* ${{ github.event.pull_request.merged_at }}\n<${{ github.event.pull_request.html_url }}|View PR>\"}}]}'
```

Comment:
Can the Author (+Reviewer) be added?

Reply:
I added the author, but users who approved the pr is not included in the event just through the api, but I don't think it's worth it.